index.html

```diff
@@ -9,14 +9,24 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
 </head>
 <body>
     <div>
         <nav class="hdr">
             <div class="logo">
                 <p>My Logo</p>
             </div>
-            <div class="bar"><img src="./images/Union.png" alt="hamburger menu"></div>
+            <div id="myButtons" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeMenu()">&times;</a>
+                <div class="overlay-content">
+                    <a href="#portfoliostart">Portfolio</a>
+                    <a href="#aboutmelink">About</a>
+                    <a href="#contactmelink">Contact</a>
+                </div>               
+            </div>
+            <span class="hamburger" onclick="openMenu()">&#9776;</span>
```

Comment:
- [ ] Your mobile menu is not working as intended. When the hamburger menu is clicked and then a menu item is selected, the mobile menu still stays visible together with the overlay, although the actual page navigates to the corresponding content of the menu item. Kindly to improve upon this aspect by ensuring that mobile menu item is not visible when a menu item is clicked or tapped.

**Mobile menu still visible after a mobile item is clicked:**
![image](https://user-images.githubusercontent.com/12745474/188551820-8581ad6c-de11-45f1-83a2-1382fca4ecc9.png)

Reply:
Thank you, fixed it!